documentation/introduction/introduction-overview.asciidoc

```diff
@@ -1,18 +1,37 @@
 ---
-title: Understanding Vaadin
+title: Overview
 order: 1
 layout: page
 ---
-= Understanding Vaadin
+= Overview
 
-Vaadin connects the Java ecosystem to your web platform. This section provides an overview of the architecture of Vaadin and introduces the key concepts in the framework.
+This chapter provides an overview of the Vaadin architecture and introduces the key concepts in the framework.
 
-== Vaadin Architecture
+== What is Vaadin?
 
-Working with front-end web technologies, such as HTML, CSS and JavaScript, can be challenging and time-consuming for Java developers. In Vaadin, all UI elements are componentized into https://www.webcomponents.org/[Web Components]. This makes development easier than ever before, because each element is decoupled and sandboxed.
+Vaadin is a user interface framework to create web apps on top of a Java backend.
+Using Vaadin, you can quickly create a modern and robust web application.
+Vaadin allows you to combine the power of Java with the capability of web browsers to create a truly class-leading user and developer experience.
+
+Vaadin consists of a large set of premade UI components and widgets that you can combine and extend in any way you like.
```

Comment:
@samie previously talked about including the term 'widget' in our docs, because many devs apparently don't know what a 'component' is. This section could clarify that those are the same thing.

Reply:
Sure, I don’t have objections to that, clarifying that “widget” is the same thing as a “web component”. I just wanted to avoid confusing readers by making it sound like components and widgets are somehow different. 

Sure, previously we had that conceptual separation between “server-side components” and “client-side widgets” (the term widget came from GWT), but now that we have “web components” as a standard term, I don’t think we benefit from ”muddying the water” with another term. So components all around IMO :)

@samie’s “client-server component” (or was it “server-client component”) is one term to bounce between us, though, if that would start to stick.